src/app/views/band-albums.html

```diff
@@ -1,5 +1,28 @@
 <div class="band-detail-container">
-    <img src="https://i.scdn.co/image/66302ca80395b6be3600d5f0ef69db9e0c43f4f5" alt="">
-    <img src="https://i.scdn.co/image/e2b11f82ba8d5aecc6486fe28225db43a003caa1" alt="">
-    <img src="https://i.scdn.co/image/3a299e917835b053f53441280262f10829bd1c8e" alt="">
-</div>
\ No newline at end of file
+    <nav class="nav">
+        <img src="/app/images/spotify-icon.png" alt="" class="logo-left" ng-click=bandAlbumsCtrl.toIndex()>
+        <div class="column-search">
```

Comment:
you should develop a directive of searchbar as it is being repeated in major views.